Classes/Domain/Model/News.php

```diff
@@ -39,14 +39,14 @@ class News extends AbstractEntity
     protected $l10nParent = 0;
 
     /**
-     * @var DateTime
+     * @var DateTime|null
```

Comment:
I am really unsure if that breaks in 10 because extbase can't get the type anymore of

Reply:
Maybe leave those changes, also imo not related to this feature, correct?